src/server/game/Achievements/AchievementMgr.cpp

```diff
@@ -2394,6 +2394,10 @@ bool AchievementMgr::CanUpdateCriteria(AchievementCriteriaEntry const* criteria,
     return true;
 }
 
+const CompletedAchievementMap& AchievementMgr::GetCompletedAchievements() {
```

Comment:
```suggestion
CompletedAchievementMap const& AchievementMgr::GetCompletedAchievements()
{
```

You can read more about the C++ standards we use here: https://www.azerothcore.org/wiki/cpp-code-standards

Reply:
thank you, is there somekind of linter I could use to enforce them? (or at least parts of it?)